components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -3173,7 +3173,8 @@ public boolean isScopeValidationEnabledForCodeAndImplicitGrant() {
     }
 
     /**
-     * Parses the AllowCrossTenantTokenIntrospection configuration.
+     * Parses the AllowCrossTenantTokenIntrospection configuration that used to allow or block token introspection
+     * from other tenants
```

Comment:
Add a full stop at the end of the sentence.

Reply:
Resolved by https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1929/commits/8919d9785ca543b01b915b630e98f69e88d5817c